### DISCUSSION TITLE
How to specify multiple import and export paths when creating FSX for Lustre filesystem in CDK

### DISCUSSION THREAD

**Discussion by yashwanthkp** [aws/aws-cdk#22606] [Q&A]
Hi,

I am creating a new FSXFileSystem for Lustre. I want to setup auto import/export from 2 different S3 buckets on the filesystem. The [LustreConfiguration in CDK](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-fsx.LustreConfiguration.html) takes only a string which lets me specify only a single bucket. How do I setup export/import on multiple buckets on creation?

```
const fsxLustreFileSystem = new fsx.LustreFileSystem(
            this,
            `FsxFileSystem`,
            {
                lustreConfiguration: {
                    deploymentType: fsx.LustreDeploymentType.PERSISTENT_2,
                    exportPath: S3Bucket.s3UrlForObject(),
                    importPath: S3Bucket.s3UrlForObject(),
                },
                storageCapacityGiB: 1200,
                vpc: vpc,
                vpcSubnet: vpc.privateSubnets[0],
                securityGroup: securityGroup,
            }
        );
```

**Comment by JulyQqq** [comment]
这是来自QQ邮箱的自动回复邮件。 
您好，您的邮件我已收到，会尽快给您回复滴~~~

**Comment by peterwoodworth** [comment]
It looks to me like CloudFormation doesn't allow for configuring multiple imports and exports, see their docs [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-fsx-filesystem.html).

CDK will be limited by what CloudFormation can provide here, if you need a way to work around this I would recommend looking into custom resoures - maybe [AwsCustomResource](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.custom_resources.AwsCustomResource.html) could help you out here if you can make a simple API call to set this

Else, I would recommend creating an issue on the [CloudFormation Coverage Roadmap](https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues) if one doesn't exist already 🙂

**Comment by JulyQqq** [comment]
这是来自QQ邮箱的自动回复邮件。 
您好，您的邮件我已收到，会尽快给您回复滴~~~
